mappings/net/minecraft/fluid/Fluid.mapping

```diff
@@ -5,13 +5,26 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 	METHOD method_15774 getBucketItem ()Lnet/minecraft/class_1792;
 	METHOD method_15775 appendProperties (Lnet/minecraft/class_2689$class_2690;)V
 	METHOD method_15776 randomDisplayTick (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;Ljava/util/Random;)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 random
+	METHOD method_15777 canBeReplaceWith (Lnet/minecraft/class_3610;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3611;Lnet/minecraft/class_2350;)Z
```

Comment:
`canReplaceWith` or `canBeReplacedWith` may be better here.

Reply:
Oops, made a typo here, I meant `canBeReplacedWith`.